ESLint error occurs in Next.js when publish

**Describe the bug**
- Try to get started Amplify Next.js according to guide document. 
- Got an error following error in step https://docs.amplify.aws/guides/hosting/nextjs/q/platform/js#adding-amplify-hosting

```bash
> next build && next export

info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
info  - Checking validity of types

Failed to compile.

./pages/index.js
62:13  Error: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element.  @next/next/no-img-element

Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amplify-next-app@0.1.0 build: `next build && next export`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amplify-next-app@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**To Reproduce**
Steps to reproduce the behavior:
1. Follow each step in this page. https://docs.amplify.aws/guides/hosting/nextjs/q/platform/js
2. Got error at `amplify publish` after `amplify add hosting`

**Expected behavior**
Success `amplify publish` command

**Desktop (please complete the following information):**
 - OS: MacOS BigSur 11.3 Apple M1
 - amplify version: 5.1.0
 - node: v14.15.1
 - yarn: 1.22.10

--- a/docs/guides/hosting/fragments/nextjs.md
+++ b/docs/guides/hosting/fragments/nextjs.md
@@ -57,6 +57,19 @@ To deploy a static app, add the `export` script to the existing `build` script:
 ```
 The build script `next build && next export` indicates that the app supports SSG pages only.`next export` allows you to export your app to static HTML, which can be run standalone without the need of a Node.js server.
 
+At last, change to eslint rule to update **.eslintrc** file. When you build a Next.js project, you will have an error since Next.js restrict an user to use `img` as `@next/next/no-img-element` eslint rule.
+
+To build successfully, append the ignore rule to the existing `.eslintrc` below `extends` block:
+
+```json
+{
+  "extends": ["next", "next/core-web-vitals"],
+  "rules": {
+    "@next/next/no-img-element": "off"
+  }
+}
+```
+
 ### Adding Amplify hosting
 
 Initialize a new Amplify project. The `Distribution Directory Path` depends on whether you are deploying a static or SSR app. For a static app, set the `Distribution Directory Path` to `out`.

